src/textual/app.py

```diff
@@ -531,6 +558,19 @@ def refresh(self, *, repaint: bool = True, layout: bool = False) -> None:
             except Exception:
                 self.panic()
 
+    def refresh_css(self, animate: bool = True) -> None:
+        """Refresh CSS.
+
+        Args:
+            animate (bool, optional): Also execute CSS animations. Defaults to True.
+        """
+        # TODO: This doesn't update variables
```

Comment:
What does this comment mean?

Reply:
It means I forgot to remote the comment! Previously the new CSS variables wouldn't updated.